src/client/util/url.ts

```diff
@@ -13,10 +13,7 @@ export async function generateShortUrl() {
   return generate(ALPHABET, LENGTH)
 }
 
-/**
- * Strips http or https from url string.
- * @param {String} url
```

Comment:
Can I check why is it still relevant, given that TypeScript will be providing the function its parameter types?

Reply:
The type isn't useful anymore but the parameter description still applies

Reply:
The relocation of types in typescript empties out the parameter description of url. But since it will be a small change, I will just add in a parameter description to make the tsdoc more meaningful.